File: python/src/integrations/query_integration/defaults.py
from ...models import QueryDefaults

DEFAULTS: QueryDefaults = QueryDefaults(
    ttl_minutes=60,
    cached=True,
    timeout_minutes=20,
    retry_interval_seconds=0.5,
    page_size=100000,
    page_number=1,
    max_age_minutes=5,
)

File: js/src/defaults.ts
import { version } from "../package.json";
import { SdkDefaults } from "./types";

export const DEFAULTS: SdkDefaults = {
  apiBaseUrl: "https://api-v2.flipsidecrypto.xyz",
  ttlMinutes: 60,
  maxAgeMinutes: 0,
  cached: true,
  dataProvider: "flipside",
  dataSource: "snowflake-default",
  timeoutMinutes: 20,
  retryIntervalSeconds: 0.5,
  pageSize: 100000,
  pageNumber: 1,
  sdkPackage: "js",
  sdkVersion: version,
};
